### ISSUE TITLE
jdbc-mssql extention is missing

### ISSUE THREAD

**Issue by jagodevreede** [quarkusio/quarkus#4188]
On the website: https://quarkus.io/guides/datasource-guide it states that:

> Quarkus provides driver extensions for:
> ....
> Microsoft SQL Server - jdbc-mssql

but  `mvn quarkus:add-extension -Dextensions=jdbc-mssql` results in:
```
[INFO] Scanning for projects...
[INFO]
[INFO] ----------------< ******** >----------------
[INFO] Building ************ 1.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- quarkus-maven-plugin:0.22.0:add-extension (default-cli) @ ******* ---
? Cannot find a dependency matching 'jdbc-mssql', maybe a typo?
```
It is also not listed on the _start coding_ page

Adding for example jdbc-h2 works. changing the dependency from h2 to mssql does work.

**Comment by jaikiran** [comment]
I think this was fixed a few days back in `master` branch https://github.com/quarkusio/quarkus/pull/4033

**Comment by jagodevreede** [comment]
Look like that will fix it, tnx!

**Comment by machi1990** [comment]
Fixed by #4033. Closing
